### PR TITLE
Added charinfo command

### PR DESCRIPTION
**Pull request by WieeRd**
Another piece of code from RoboDanny :)
Great for finding out unicode codepoint of default emojis.

### CODE REVIEW

**Review by TAG-Epic — CHANGES_REQUESTED** [review]
This should be moved to its own file and have r.dannys license on the top

**Comment by WieeRd** [comment]
You mean I should create new Cog/Extension with this command in it, and paste the license.txt of R. Danny on top?

**Comment by TAG-Epic** [comment]
Yeah.

**Comment by WieeRd** [comment]
Done.
R. Danny's license is huge though

**Comment by TAG-Epic** [comment]
Actually wait no there should be a shorter license at the top of r.danny forks

**Comment by WieeRd** [comment]
Older version of R. Danny had much shorter MIT license, 
while current rewrite branch has humongous Mozilla Public License Version 2.0.
`?charinfo` command did exist when R. Danny was using MIT license.
Are you telling me to paste that license? (please make it clear this time)

**Comment by ooliver1** [comment]
The shorter license usually compliments a license that comes with the software, which is the rdanny license file

**Comment by WieeRd** [comment]
Can someone just give me the exact link to the "shorter license"? I have no idea where to find it.

**Comment by TAG-Epic** [comment]
Ah true @ooliver1

File: cogs/rtfm.py [hunk 1]
```diff
@@ -2,6 +2,7 @@
 import os
 import re
 import zlib
+import unicodedata
```
Comment (TAG-Epic) [inline cogs/rtfm.py:5]: Unused import

**Review by TAG-Epic — CHANGES_REQUESTED** [review]

File: cogs/rtfm.py (hunk 1 above)
Comment (WieeRd) [reply cogs/rtfm.py:5]: Oh sorry forgot to remove it after moving the command to etc.py.

**Comment by WieeRd** [comment]
So what exactly should I do with the license?? I'm confused.

**Comment by TAG-Epic** [comment]
Yeah just leave it as is

**Comment by WieeRd** [comment]
Alright. Fixed unused import as well. I'll wait for the merge.

**Review by TAG-Epic — APPROVED** [review]

**Pull request merged**